modules/prometheus-exporter_oracle/conf/09-stby.yaml

```diff
@@ -0,0 +1,19 @@
+module: oracle
+name: "gap in standby database replication"
+id: STBY_Replication
+
+transformation: true
+aggregation: ".mean(by=['dbname','dbtype','hostname'])"
+
+signals:
+  signal:
+    metric: oracledb_STBY_Replication_count 
+    rollup: latest
+
+rules:
+  critical:
```

Comment:
I would say `major` here, this is not related to direct/immediate availability problem